modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -8,12 +8,19 @@ module Veterans
       class IntentToFileController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
-        ITF_TYPES = %w[compensation pension burial].freeze
-
+        # GET to fetch active intent to file by type
+        #
+        # @return [JSON] ITF record
         def type
+          itf_types = ClaimsApi::V2::ParamsValidation::IntentToFile::IntentToFileValidator.itf_types
```

Comment:
seeing this error when running this line in the console. 
```
NameError (uninitialized constant ClaimsApi::V2::ParamsValidation)
```

Reply:
referenced below (https://github.com/department-of-veterans-affairs/vets-api/pull/11411/files#r1056534625) but may just set `itf_types` to this and we could remove the `self.itf_types` in the comment linked:
```
ClaimsApi::IntentToFile::ITF_TYPES_TO_BGS_TYPES.keys
 => ["compensation", "survivor", "pension"]
 ```
 
 Not sure if rubocop would like that, but?